pages/Category/components/Content/index.jsx

```diff
@@ -12,12 +12,11 @@ import connect from './connector';
  * @param {Object} props.categoryId The category id.
  * @returns {JSX}
  */
-const CategoryContent = ({ categories, categoryId, hasProducts }) => (
+const CategoryContent = (
+  { categories, categoryId, hasChildren, hasProducts }
```

Comment:
Please don't break the line before and after the curly bracket. Those have to go into the same line than the `(` and `)`. Stick with the code style we have. And btw: your eslint should have complained about that! Please check your setup.